modules/price/src/Element/Price.php

```diff
@@ -27,16 +27,16 @@
  *
  * @FormElement("commerce_price")
  */
-class Price extends Textfield {
+class Price extends FormElement {
 
   /**
    * {@inheritdoc}
    */
   public function getInfo() {
     $class = get_class($this);
     return [
-      '#size' => 10,
-      '#maxlength' => 128,
+      '#amount_size' => 10,
```

Comment:
I didn't see any other patterns like that, here's the check in question `\Drupal\Core\Form\FormValidator::performRequiredValidation`.

I'll just unset after we pass to amount